docs/source/authentication.rst

```diff
@@ -7,8 +7,8 @@ That means that any authentication method supported by an installed PAS Plugin s
 
 For example, to authenticate using HTTP basic auth, you'd set an ``Authorization`` header:
 
-.. literalinclude:: _json/siteroot.req
-   :language: http
+..  http:example:: curl httpie python-requests
+    :request: _json/siteroot.req
 
 HTTP client libraries usually contain helper functions to produce a proper ``Authorization`` header for you based on given credentials.
```

Comment:
I would remove also the following paragraphs until the JSON Web Tokens (JWT) section, because the explanation is already covered by the example.

Reply:
@erral  I don't think that would harm anyone. besides, it's not even wrong information. I guess we should leave it like that... 🙂

Reply:
I said that because when using the `http:example` tag and you build the documentation a nice tabbed interface is shown with all the examples with python requests, curl, httpie and plain http.

That's why we are using it all through the documentation: to write one HTTP request and render it using multiple renderers (python-requests, httpie, curl). 

If we keep the following paragraphs we are repeating the information already given by the `http-example`.

You can check how the built documentation looks like running `./bin/sphinxbuilder` in your buildout and then looking at the documents created on `docs/html`

Reply:
Thanks! @erral